src/components/select/_select.scss

```
@@ -196,3 +197,204 @@
     display: none;
   }
 }
+
+@mixin select--experimental {
+  .#{$prefix}--select {
+    @include reset;
+    position: relative;
+    display: flex;
+    flex-direction: column;
+  }
+
+  .#{$prefix}--select-input {
+    @include font-family;
+    @include typescale('zeta');
+    height: rem(40px);
+    appearance: none;
+    display: block;
+    width: 100%;
+    padding: 0 $spacing-2xl 0 $spacing-md;
+    color: $text-01;
+    background-color: $field-01;
+    border: none;
+    box-shadow: 0 1px 0 0 $ui-05;
+    order: 2;
+    border-radius: 0;
+    cursor: pointer;
+    border-bottom: 1px solid transparent;
+
+    // Hide default select arrow in IE10+
+    &::-ms-expand {
+      display: none;
+    }
+
+    & ~ .#{$prefix}--label {
+      order: 1;
+    }
+
+    &:focus {
+      outline: none;
+      box-shadow: none;
+      border: 2px solid $brand-01;
+    }
```

```suggestion
    &:focus {
      outline: 2px solid $focus;
      box-shadow: none;
    }
```

Using a border is causing a shift in width, would be good to swap to outline.